crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -72,10 +131,19 @@ def services
     end
 
     def backup
+      @current_step = 8
       if request.post?
         respond_to do |format|
-          format.html do
-            redirect_to nodes_upgrade_url
+          begin
+            @service_object.finalize_openstack_shutdown
```

Comment:
Please add `Openstack::Upgrade.unset_db_synced` here... at least until we find out better place.